package.json

```diff
@@ -4,14 +4,14 @@
     "description": "",
     "main": "index.js",
     "scripts": {
-        "build": "tsc",
+        "build": "tsc -p tsConfig.build.json",
         "commit": "commit",
         "dev": "nodemon ./src/index.ts",
         "format": "prettier ./src/**/*.ts --write",
         "lint": "eslint ./src/**/*.ts --fix",
         "prepare": "husky install",
         "start": "node dist/index.js",
-        "test": "echo \"Error: no test specified\" && exit 1"
+        "test": "jest --detectOpenHandles"
```

Comment:
And I think you no longer need this flag as you solved the problem. Just `jest` will be fine 😊